components/heading/src/overrides/brandHeading.js

```diff
@@ -25,12 +27,29 @@ const BrandHeading = forwardRef(({ state: { tag: Tag, size }, ...rest }, ref) =>
 
 const brandHeadingStyles = (_, { size }) => {
 	const { PACKS, TYPE } = useBrand();
+	const mq = useMediaQuery();
+
+	const tokens = PACKS.typeScale.brandFont;
+	const sizeMax = Object.keys(tokens).reduce((a, b) => (tokens[a] > tokens[b] ? a : b));
```

Comment:
This is doing a comparison on the token size object which doesn't make sense. Should just be 
`Math.max(...Object.keys(tokens))` to find the max key value